Sources/CustomDump/Conformances/SwiftUI.swift

```diff
@@ -1,4 +1,4 @@
-#if canImport(SwiftUI)
+#if canImport(SwiftUI) && DEBUG
```

Comment:
This is one thing I don't understand. `Animation` and `LocalizedStringKey` can't be found in release mode.